.github/workflows/release.yaml

```diff
@@ -9,14 +9,12 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-      - name: Fetch history
-        run: git fetch --prune --unshallow
-
-      - name: Install deps
-        run: .ci/github/install-release-deps.sh
+      - name: Run hack/ci/github/setup/release.sh
```

Comment:
I would prefer setting the name to what te script actually does.

Reply:
It sets up the environment needed for the release.yaml github workflow.